ladybug_geometry/geometry2d/line.py

```diff
@@ -72,6 +73,10 @@ def length(self):
         """The length of the line segment."""
         return self.v.magnitude
 
+    def to_array(self):
+        """ To nested list """
+        return (self.p1.to_array(), self.p2.to_array())
```

Comment:
Can we add this same type of method to the `LineSegment3D` class?  It seems odd that we would have it here and not in 3D.  Also, it would be good to clarify in the docstring for this method that the list has two nested lists that represent the two endpoints (not the first point and a vector).